deliverables/ruby_monk_problems/random_select.rb

```diff
@@ -1,3 +1,7 @@
 def random_select(array, n)
-  # your code here
+  ans = []
```

Comment:
Looks like a lot of copy pasta.  Did you just change some variable names in some of these?